client/libs/core-state/src/lib/media/media.effects.ts

```diff
@@ -0,0 +1,78 @@
+import { Injectable } from '@angular/core';
+
+import { createEffect, Actions } from '@ngrx/effects';
+import { DataPersistence } from '@nrwl/angular';
+import { map } from 'rxjs/operators';
+
+import * as fromMedia from './media.reducer';
+import * as MediaActions from './media.actions';
+import { MediaService, Media } from '@sb/core-data';
+
+@Injectable()
+export class MediaEffects {
+  loadMedia$ = createEffect(() =>
+    this.dataPersistence.fetch(MediaActions.loadMedia, {
+      run: (
+        action: ReturnType<typeof MediaActions.loadMedia>,
+        state: fromMedia.MediaPartialState
+      ) => {
+        return this.mediaService.all().pipe(map((res: Media[]) => MediaActions.loadMediaSuccess({ media: res})));
+      },
+
+      onError: (action: ReturnType<typeof MediaActions.loadMedia>, error) => {
+        console.error('Error', error);
+      }
+    })
+  );
+
+  addMedia$ = createEffect(() =>
+    this.dataPersistence.fetch(MediaActions.createMedia, {
```

Comment:
Please use this.dataPersistence.pessimisticUpdate for add, update and delete effects.
https://nx.dev/angular/guides/misc-data-persistence#pessimistic-updates